pkg/upgradepatch/hc.cr.json

```diff
@@ -22,7 +22,8 @@
       "completionTimeoutPerGiB": 150,
       "progressTimeout": 150,
       "allowAutoConverge": false,
-      "allowPostCopy": false
+      "allowPostCopy": false,
+      "maxHotplugRatio": 4
```

Comment:
This file is auto generated by `make gogenerate`. Do not manually edit it.

Reply:
Also, it seems that this field belong to the new `LiveUpdateConfiguration` field, and not to this field  (`LiveUpdateConfiguration`).

Reply:
This is a leftover, I will remove it :+1: